modules/management/locals.tf

```diff
@@ -185,7 +185,7 @@ locals {
         Deploy-ASC-Defender = {
           emailSecurityContact                = local.settings.security_center.config.email_security_contact
           logAnalytics                        = local.log_analytics_workspace_resource_id
-          ascExportResourceGroupName          = "${local.root_id}-asc-export"
+          ascExportResourceGroupName          = local.azurerm_log_analytics_workspace.resource_group_name
```

Comment:
Unfortunately we cannot just change this value as suggested. Please see my more detailed explanation on my review.

Reply:
Further to our conversation today, please omit this change from the PR and we will look at a way we can provide this functionality whilst leaving the "defaults" in place for our greenfield customers. I will also discuss with our team as to whether we actually need this to be in a dedicated Resource Group given we only deploy a single ASC export policy.

This decision will then drive the source Enterprise-scale configuration, and we will look to include the configuration of this Resource Group and `ExportToWorkspace` security automation as part of the resources managed by Terraform rather than policy.

> We will continue to use policy to monitor compliance.

Reply:
Commit reverted